### ISSUE TITLE
Profile Descriptions Not Displayed Properly

### ISSUE THREAD

**Issue by lionirdeadman** [AniTrend/anitrend-app#52]
Currently the profile descriptions don't render as intended and this should be address for consistency's sake.

**Comment by wax911** [comment]
I think we already discussed this as there will be no real solution for how the custom html is rendered, each solution has it's own implications and after all the attempts I've tried in the past the current implementation is the best way forward

**Comment by lionirdeadman** [comment]
Right, I'll ask Josh if he can give us Markdown so you can leave this issue open until something is done about this.
